doc/install/fast-install-and-first-run.rst

```diff
@@ -23,6 +23,21 @@ For another way to run IVRE easily (probably even more easily), see
 
 Install
 -------
+NB: if you are running Debian stable, the dokuwiki package has been
```

Comment:
I **think** this is no longer true. Actually I have forgotten to update this page when I fixed the Docker image in c1c94ce1654d2b20fe47b45e471b8c81318d5cd6.

Reply:
So this can be removed?

Reply:
Yes